docs/doc-guidelines.md

```diff
@@ -0,0 +1,83 @@
+
+If you're new to writing or editing ownCloud documentation, we recommend the [Getting Started Guide](https://github.com/owncloud/docs/blob/master/docs/getting-started.md) and the [Building the Documentation](https://github.com/owncloud/docs/blob/master/docs/build-the-docs.md) so you get a good understanding what awaits you. (Just don't let it discourage you!). Keep the [Best Practises Guide](https://github.com/owncloud/docs/blob/master/docs/best-practices.md) collection in mind.
+
+To contribute to the documentation by providing content or fixing issues in the existing documentation:
+
+- Clone the respective repository.
+- Create a local branch.
+- Edit an existing file or add a new one.
+
+Before pushing the file to Github, please take care of the following:
+
+1. Our documentation language is American English. If in doubt how things are written, consult a dictionary or use a spell-checker. If in doubt about your English language skills in general, request a language review from `@EParzefall` or `@phil-davis` when you create a pull request on Github.
+2. While writing or editing, please follow the [Style Guide](https://github.com/owncloud/docs/blob/master/docs/style-guide.md).
+3. [Build the documentation](https://github.com/owncloud/docs/blob/master/docs/build-the-docs.md) and check the layout.
+
+Commit all change files and push the branch to GitHub. Now, log in to Github and find the branch you pushed, create a pull request and follow the rules below to make the process quick and easy for everyone:
+
+1. 4 Eyes Principle (4EP)
+
+   Each pull request (PR) on Github must pass the 4EP. This means that each PR needs an approval before the PR gets merged into the master branch. Add technical reviewers for peer reviews or anyone else who can give you valuable feedback. Author and reviewers share the responsibility of driving the process forward. A reviewer should merge after approval if no more steps or reviews are needed. Otherwise, the reviewer should point out any next steps needed (like e.g. backports), ask for more reviews or ask the author to take action (e.g. please merge after extra reviews or steps). Particularly if you don't want to take care of everything else in the approval and merge process, like backports, add one the following persons as reviewers: `@eparzefall`, `@mmattel`, `@phil-davis`.
+
+   Exceptions to the 4EP must be documented in the PR description and can be made if:
+
+   1. language violations require immediate action,
+   2. the docs web site is not usable, partially or in full, without an emergency docs PR.
+
+2. Security and Dependency Bumps
+
+   These PRs are usually made by the bot but can also be created manually. SDB are related to the build process tools. The 4EP applies. The approver must take responsibility not to break the existing setup. Mandatory: add `@phil-davis` as reviewer.
```

Comment:
Hmmm - I don't need to review all dependency PRs - it's just that I wake up first in the morning and usually see the dependabot PRs first.

Reply:
You have been the "master" of SDBs in the past 😃 But I can add myelf to the list to relax this. I just want to avoid a silent merge which could cause build troubles. SDBs are too important to be merged unnotified and I know that you are a person of knowledge and trust 🙏 . Let me know if I should add myself...

Reply:
@mmattel please put yourself here also - add mmattel or phil-davis as reviewer.

Reply:
done